content/blog/getting-started-with-hpe-greenlake-api-for-backup-and-recovery.md

```diff
@@ -0,0 +1,591 @@
+>                 "protectionStoreType": "ON_PREMISES"
+>             },
+>             "type": "BACKUP"
+>         },
+>         {
+>             "id": “<cloud-protection-id>”,
+>             "schedules": [
+>                 {
+>                     "scheduleId": 3,
+>                     "sourceProtectionScheduleId": 2,
+>                     "name": "HPE_Cloud_Protection_Store_3",
+>                     "schedule": {
+>                         "recurrence": "DAILY",
+>                         "repeatInterval": {
+>                             "every": 2
+>                         },
+>                         "startTime": "00:00"
+>                     },
+>                     "expireAfter": {
+>                         "unit": "WEEKS",
+>                         "value": 1
+>                     },
+>                     "namePattern": {
+>                         "format": "HPE_Cloud_Protection_Store_{DateFormat}"
+>                     }
+>                 }
+>             ],
+>             "protectionStoreInfo": {
+>                 "id": "cloud-protection-store-id",
+>                 "name": "Cloud_CDS-TPM-PSGno1_USA, North Virginia",
+>                 "type": "backup-recovery/protection-store",
+>                 "region": "USA, North Virginia",
+>                 "resourceUri": "/backup-recovery/v1beta1/protection-stores/25611df9-xxxx-xxxx-xxxx-aa3a4887869f",
+>                 "protectionStoreType": "CLOUD"
+>             },
+>             "type": "CLOUD_BACKUP"
+>         }
+>     ],
+>     "createdAt": "2024-04-05T03:28:03.000000Z",
+>     "createdBy": {
+>         "id": "<user-id>",
+>         "name": "ronald.dharma@hpe.com"
+>     },
+>     "generation": 1,
+>     "resourceUri": "/backup-recovery/v1beta1/protection-policies/f572ce6e-xxxx-xxxx-xxxx-530284bf2bc4",
+>     "consoleUri": "/backup-and-recovery/protection-policies/f572ce6e-xxxx-xxxx-xxxx-530284bf2bc4",
+>     "applicationType": "VMWARE",
+>     "type": "backup-recovery/protection-policy",
+>     "updatedAt": "2024-04-05T03:28:03.000000Z"
+> }
+> ```
+
+## Applying protection policy to a Virtual Machine
+
+Next in the list of use cases for this blog post, I followed the progression for the day one activities. The next one is to protect a virtual machine which is applying this protection policy to a virtual machine (or any other assets). The steps required to apply the protection policy against a virtual machine are listed below:
+
+1. I obtained the values from `virtual machine id, name, and type` keys which were going to be used as the key-pair values required for the key `assetInfo` as shown in below figures. To obtain those, I used the HPE GreenLake [API](https://developer.greenlake.hpe.com/docs/greenlake/services/virtualization/public/openapi/virtualization-public-v1beta1/tag/virtual-machines/) for virtualization to discover the detail information of a virtual machine `“0-Linux-Demo-VM02”`. The API used for this: 
```

Comment:
Change "which" to "that" in the first sentence.
Should "for virtualization" be "for Virtualization"?